universal-application-tool-0.0.1/app/auth/ProfileFactory.java

```diff
@@ -35,17 +31,18 @@ public UatProfile wrapProfileData(UatProfileData p) {
   }
 
   private UatProfileData create(Roles role) {
-    UatProfileData p = new UatProfileData(clock);
+    // Not sure if this is bad to do
```

Comment:
This should be good! We want to maintain behavior, which calling the new constructor does (once you add `super()`, see comment below)